inc/admin/class-catalog-list-table.php

```diff
@@ -236,7 +228,7 @@ function get_columns() {
 		];
 
 		for ( $i = 1; $i <= Catalog::MAX_TAGS_GROUP; ++$i ) {
-			$columns[ "tag_{$i}" ] = ! empty( $profile[ "pb_catalog_tag_{$i}_name" ] ) ? esc_html( strip_tags( $profile[ "pb_catalog_tag_{$i}_name" ] ) ) : __( 'Tag', 'pressbooks' ) . " $i";
+			$columns[ "tag_{$i}" ] = ! empty( $profile[ "pb_catalog_tag_{$i}_name" ] ) ? esc_html( wp_strip_all_tags( $profile[ "pb_catalog_tag_{$i}_name" ] ) ) : __( 'Tag', 'pressbooks' ) . " $i";
```

Comment:
Replace `strip_tags` with `wp_strip_all_tags` in every occurence within Pressbooks, as recommended by linter.